src/s3Export.ts

```diff
@@ -116,6 +116,7 @@ export async function storeResultInS3(
         config.region
       }/y=${new Date().getFullYear()}/${new Date().getTime()}.json`,
       Body: JSON.stringify(metricsData),
+      ACL: 'bucket-owner-full-control'
```

Comment:
What does this do?

Reply:
grants the bucket owner full access to objects.  

Prevents objects written to the bucket that the bucket owner cannot read.